### PR TITLE
Add Pix full logo

### PR DESCRIPTION
**Pull request by pavel-craciun-sumup**
Addresses SA-53794

## Purpose

To add the Pix coloured logo provided by Brazilian team and to use it as a Card Schemes option on the Website

## Approach and changes

Added 24px coloured Pix logo to the Payment Method category

## Definition of done

* [ ] Development completed
* [ ] Reviewers assigned
* [ ] Unit and integration tests
* [ ] Meets minimum browser support
* [ ] Meets accessibility requirements

### CODE REVIEW

File: packages/icons/web/v2/pix_full_24.svg [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        <stop offset="0" style="stop-color:#f9fafb"/>
+        <stop offset=".247" style="stop-color:#f7f7f8"/>
+        <stop offset=".403" style="stop-color:#eeeff0"/>
+        <stop offset=".535" style="stop-color:#e0e0e2"/>
+        <stop offset=".652" style="stop-color:#cdcbce"/>
+        <stop offset=".76" style="stop-color:#b3b0b5"/>
+        <stop offset=".862" style="stop-color:#948f95"/>
+        <stop offset=".956" style="stop-color:#706971"/>
+        <stop offset="1" style="stop-color:#5c545d"/>
+    </radialGradient>
```
Comment (connor-baer) [inline packages/icons/web/v2/pix_full_24.svg:15]: This radial gradient isn't visible in the SVG and can probably be removed.

File: packages/icons/web/v2/pix_full_24.svg [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<svg version="1.1" id="pix_full_24_svg__Layer_1" xmlns="http://www.w3.org/2000/svg" x="0" y="0" viewBox="0 0 600 600" style="enable-background:new 0 0 600 600" xml:space="preserve">
```
Comment (connor-baer) [inline packages/icons/web/v2/pix_full_24.svg:1]: The SVG should be 24px tall and maintain the original aspect ratio of the logo. There should be no whitespace around the graphic.

**Review by connor-baer — CHANGES_REQUESTED** [review]
If you don't mind, I will commit an optimized version of this asset to this branch.

**Review by connor-baer — APPROVED** [review]

**Comment by pavel-craciun-sumup** [comment]
Thanks @connor-baer, didn't know that, just used provided svg

**Comment by connor-baer** [comment]
[Looking good](https://oss-circuit-ui-git-feature-pix-full-icon-sumup.vercel.app/?path=/docs/features-icons--page):

Screenshot 2022-01-27 at 10 32 37

File: .changeset/full-pix-24.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+'@sumup/icons': patch
```
Comment (connor-baer) [inline .changeset/full-pix-24.md:2]: Didn't spot this before 🙈 This is a new feature, so it should be a minor.

```suggestion
'@sumup/icons': minor
```

**Pull request merged**